src/components/Discussions/DiscussionCard.tsx

```diff
@@ -388,11 +388,15 @@ export const DiscussionCard = (props: IProps) => {
   };
 
   useEffect(() => {
-    if ((onAddComplete !== undefined) !== editing) {
+    if (Boolean(onAddComplete) !== editing) {
       handleReset();
       setEditing(onAddComplete !== undefined);
       if (!onAddComplete) setEditSubject(discussion.attributes?.subject);
```

Comment:
Should we just remove this check and always do the setEditSubject?

Reply:
simplified subject setting